### PR TITLE
Add gem reform and gem virtus

### PR DESCRIPTION
**Pull request by gwmoura**
Add new gems:

* gem reform
* gem virtus

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -13,3 +13,4 @@ gemspec
 # To use a debugger
 # gem 'byebug', group: [:development, :test]
 
+gem 'reform', github: 'apotonick/reform', branch: 'reform-2'
```
Comment (fernandes) [inline Gemfile]: reform 2 was released, you can use the release, any special reason to use github master?

Comment (gwmoura) [inline Gemfile]: no, we get this reference from mayhem, for not break compatibility. I go to change for branch master

Comment (fernandes) [inline Gemfile]: I mean, reform was released ( https://rubygems.org/gems/reform/versions/2.0.4 ), we can use this version

Comment (gwmoura) [inline Gemfile]: Oh! Ok `gem 'reform', '~> 2.0.4'` I go set

**Pull request merged**